addons/web_editor/static/src/js/widgets.js

```diff
@@ -496,7 +496,24 @@ var getCssSelectors = function(filter) {
     }
     var sheets = document.styleSheets;
     for(var i = 0; i < sheets.length; i++) {
-        var rules = sheets[i].rules || sheets[i].cssRules;
+        var rules;
+        if (sheets[i].rules) {
+            rules = sheets[i].rules;
+        } else {
+            //try...catch because Firefox not able to enumerate document.styleSheets[].cssRules[] for cross-domain stylesheets.
+            try {
+                if (!sheets[i].cssRules) {
+                    continue;
+                }
```

Comment:
shorter if you set "rules = sheets[i].cssRules;" here and remove the condition and the line 514